entrypoint.sh

```diff
@@ -1,28 +1,41 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 if [ "$PHP_FPM_USER_ID" != "" ]; then
-    (>&2 echo "Warning: PHP_FPM_USER_ID is deprecated. Please use WWW_DATA_USER_ID instead.")
-    WWW_DATA_USER_ID=$PHP_FPM_USER_ID
+    usermod -u $PHP_FPM_USER_ID nobody
 fi
 
-# Set the uid that www-data will run as if one was specified
-if [ "$WWW_DATA_USER_ID" != "" ]; then
-    usermod -u $WWW_DATA_USER_ID www-data
-fi
```

Comment:
Why did you remove the `WWW_DATA_USER_ID` and restore the deprecated `PHP_FPM_USER_ID`?

Reply:
my bad. I removed both while testing and forgot to add back.